### PR TITLE
Us 978 tableur corrections diverses

### PR DESCRIPTION
**Pull request by dbraquart**
This US contains various fixes, then please refer to US description.

A special point of attention: to reset the filter value (not the regexp itself, but the string/input value), I had to use a "useRef": please tell me if there is a better way.

### CODE REVIEW

**Review by Meklo — APPROVED** [review]
Code Review: OK
Tests: OK

File: src/components/network/network-table.js [hunk 1]
```diff
@@ -830,9 +910,8 @@ const NetworkTable = (props) => {
         return generatedTableColumns;
     }
 
-    function renderTable() {
+    function renderTable(rows) {
```
[the remark this answers is not in the document]
Comment (dbraquart) [reply src/components/network/network-table.js:913]: rows.length was needed **before** renderTable(), to disable the CSV export button when there is no data/rows.
So I compute rows, then I can disable the CSV button if the size is 0, and then I pass the rows to renderTable().

File: src/components/network/network-table.js [hunk 2]
```diff
@@ -360,6 +406,21 @@ const NetworkTable = (props) => {
         ]
     );
 
+    const onTabChange = useCallback(
+        (index) => {
+            console.debug('onTabChange from ' + tabIndex + ' to ' + index);
```
Comment (YenguiSeddik) [inline src/components/network/network-table.js:411]: I don't think we need to log the index of the tab when we change it

Comment (dbraquart) [reply src/components/network/network-table.js:411]: ok, I will remote it

**Review by YenguiSeddik — APPROVED** [review]
Code review: OK
Tests: OK

**Pull request merged**